Engine_Revit_UI/Query/BHoMUnitType.cs

```diff
@@ -0,0 +1,85 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2018, the respective contributors. All rights reserved.
```

Comment:
Copyright should be 2019

Reply:
Shouldn't we ctrl+h whole repo? Or shouldn't some bot do that for us?

Reply:
Only the files which change need updating - CI checks will pick it up in the future

Reply:
Fixed.